src/Silverback.Integration.MQTT/Messaging/Configuration/Mqtt/MqttClientConfig.cs

```diff
@@ -229,6 +230,16 @@ public TimeSpan KeepAlivePeriod
             set => _clientOptions.TopicAliasMaximum = value;
         }
 
+        /// <summary>
+        ///     Gets or sets a callback that is called when the <see cref="IMqttClient"/> connected to the broker.
+        /// </summary>
+        public Action<MqttClientConfig>? OnConnected { get; set; }
```

Comment:
Maybe `ClientConnectedCallback` would be a better name for this in the context of the `MqttClientConfig`?

Same for `OnDisconnecting`.

Reply:
I'd prefer the `Handler` suffix, similar to https://github.com/BEagle1984/silverback/blob/master/src/Silverback.Integration.Kafka/Messaging/KafkaEvents/KafkaConsumerEventsHandlers.cs.
I also believe that this properties shouldn't be in the `MqttClientConfig` class but encapsulated in their own class and referenced in the top level of the endpoint. The `MqttClientConfig` class is meant to configure the underlying client and this is an extra functionality added by Silverback on top of it, so it doesn't really belong in there. What do you think?